src/redux/books/books.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+const Books = () => {
```

Comment:
- [ ] The **`redux`** directory is meant for redux-related configurations, not components. At this point in the project, this file should be empty. Kindly move this **Books** component into the `src/component` folder as that is where your components should live.
- [ ] The **`Books`** component is intended for listing books using the **`Book`** component and the component for adding a new book.